server/test/unit/server/managers/repo/test_sync.py

```diff
@@ -75,10 +75,12 @@ def clean(self):
     def test_queue_sync(self, mock_sync_task):
         repo_id = 'foo'
         overrides = {'baz': 1}
-        self.sync_manager.queue_sync_with_auto_publish(repo_id, overrides)
+        scheduled_call_id = None
```

Comment:
This should probably take on a more meaningful value than 'None'. The kwarg defaults to None so forcing the default value is probably not as good as setting it to a string ie: `1234`